### PR TITLE
move mnemonic menu below input

### PR DESCRIPTION
**Pull request by wolverineks**

### CODE REVIEW

File: legacy/components/WalletInit/RestoreWallet/RestoreWalletScreen/MnemonicInput.js [hunk 1]
```diff
@@ -87,8 +86,12 @@ const MnemonicWordInput = React.forwardRef(({id, onSelect}: MnemonicWordInputPro
     onSelect(normalizeText(word))
   }
 
+  const rowHeight = 48
```
Comment (stackchain) [inline legacy/components/WalletInit/RestoreWallet/RestoreWalletScreen/MnemonicInput.js]: I'd consider moving it to the top and use capital letters

File: legacy/components/WalletInit/RestoreWallet/RestoreWalletScreen/MnemonicInput.js [hunk 2]
```diff
@@ -112,9 +115,9 @@ const MnemonicWordInput = React.forwardRef(({id, onSelect}: MnemonicWordInputPro
         setWord('')
       }}
     >
-      <ScrollView style={{maxHeight: 48 * 3.5 /* 3.5 rows */}} keyboardShouldPersistTaps={'always'}>
+      <ScrollView style={{maxHeight: rowHeight * 3.5}} keyboardShouldPersistTaps={'always'}>
```
Comment (stackchain) [inline legacy/components/WalletInit/RestoreWallet/RestoreWalletScreen/MnemonicInput.js]: Perhaps transform the magic number into another constant on top?

**Comment by stackchain** [comment]
When testing, the two last lines of words the popup is hidden under the keyboard, could you roll the screen up when the user are on the last 2/3 rows?

File: legacy/components/WalletInit/RestoreWallet/RestoreWalletScreen/MnemonicInput.js (hunk 1 above)
Comment (wolverineks) [reply legacy/components/WalletInit/RestoreWallet/RestoreWalletScreen/MnemonicInput.js]: my preference is colocate data so its easier to change and doesnt get in the way of unrelated code

Comment (wolverineks) [reply legacy/components/WalletInit/RestoreWallet/RestoreWalletScreen/MnemonicInput.js]: i can go either way on all caps.
but if there is too much then it gets really noisy (this repo has waaaaaay too much all caps constants)

**Comment by wolverineks** [comment]
> When testing, the two last lines of words the popup is hidden under the keyboard, could you roll the screen up when the user are on the last 2/3 rows?

try it now

**Comment by stackchain** [comment]
Working beautifully

https://user-images.githubusercontent.com/30806844/139351686-44502c49-fd1f-452b-ad8d-917fe852f6e2.mp4

**Review by stackchain — APPROVED** [review]

**Comment by zuzunker** [comment]
> Working beautifully

Sweeeeeet 👌 Great rework, @wolverineks! 🙌

**Pull request merged**